docs/client/models.md

```diff
@@ -0,0 +1,20 @@
+# <img align="center" src="../images/logo.png">  Accessing Models and Enums
+
+## General
+
+Models and enums are generated in the `models` namespace. So, say you are using package `@azure/pets`. To access model `Dog`, you would use the following code
+snippet
+
+```js
+import { Dog } from "@azure/pets.models";
```

Comment:
`models` are listed in the root module so `import { Dog } from "@azure/pets"` should be enough. However, it is interesting to see this suggested and I would like to learn more about your thinking around using `.models`. Was it so that IntelliSense shows only modules instead of suggesting the whole set of publicly exposed things?

/cc: @xirzec

Reply:
Unfortunately, we don't have a robust way of supporting different package entry points. Importing `pets.models` can run into several issues especially with bundlers.
```suggestion
import { Dog } from "@azure/pets";
```

Reply:
ah i see. I went with models bc it's in the `models` module, and i think that's how otherlanguages are doing it. But sinceyou guys do'nt have. a robust way of supportig diff package entrypoints, makes sese to just import from azure/pets. Thanks!